WalletWasabi.Fluent/ViewModels/Wallets/Send/AdvancedSendOptionsViewModel.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Globalization;
+using System.Reactive;
+using System.Reactive.Linq;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Fluent.Validation;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Send
+{
+	[NavigationMetaData(Title = "Advanced")]
+	public partial class AdvancedSendOptionsViewModel : DialogViewModelBase<Unit>
+	{
+		private readonly TransactionInfo _transactionInfo;
+
+		[AutoNotify] private string _customFee;
+
+		public AdvancedSendOptionsViewModel(TransactionInfo transactionInfo)
+		{
+			_transactionInfo = transactionInfo;
+
+			_customFee = transactionInfo.CustomFeeRate != FeeRate.Zero
+				? transactionInfo.CustomFeeRate.SatoshiPerByte.ToString(CultureInfo.InvariantCulture)
+				: "";
+
+			EnableBack = false;
+			SetupCancel(enableCancel: true, enableCancelOnEscape: true, enableCancelOnPressed: true);
+
+			this.ValidateProperty(x => x.CustomFee, ValidateCustomFee);
+
+			var nextCommandCanExecute =
+				this.WhenAnyValue(x => x.CustomFee)
+					.Select(_ =>
+					{
+						var noError = !Validations.Any;
+						var somethingFilled = CustomFee is not null or "";
+
+						return noError && somethingFilled;
+					});
+
+			NextCommand = ReactiveCommand.Create(OnNext, nextCommandCanExecute);
+		}
+
+		private void OnNext()
+		{
+			_transactionInfo.CustomFeeRate =
+				decimal.TryParse(CustomFee, NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture, out var customFee)
+					? new FeeRate(customFee)
+					: FeeRate.Zero;
+
+			Close(DialogResultKind.Normal, Unit.Default);
+		}
+
+		private void ValidateCustomFee(IValidationErrors errors)
+		{
+			var customFeeString = CustomFee;
+
+			if (customFeeString is null or "")
+			{
+				return;
+			}
+
+			if (!decimal.TryParse(customFeeString, NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture, out var value))
+			{
+				errors.Add(ErrorSeverity.Error, "The entered fee is not valid.");
+				return;
+			}
+
+			if (value == decimal.Zero)
+			{
+				errors.Add(ErrorSeverity.Error, "Cannot be 0.");
```

Comment:
Fee rate can't be less than 1, AFAIK that is the minimum replay fee.
If we don't do this we will get an exception when we proceed with making the transaction.

```suggestion
			if (value < decimal.One)
			{
				errors.Add(ErrorSeverity.Error, "Cannot be less than 1 sat/vByte.");
```

Reply:
ACK